rollup.config.js

```diff
@@ -27,7 +27,7 @@ function rollupConfig({ formats, prod }) {
             exclude: ['test/*'],
             typescript: require('typescript')
         }),
-        prod !== undefined && replace({ 'process.env.NODE_ENV': replaceToken }),
+        prod !== undefined && replace({ 'process.env.NODE_ENV': replaceToken, preventAssignment: true }),
         prod && terser()
     ].filter(Boolean);
```

Comment:
Filtering is not needed anymore in the latest versions of rollup. 

```suggestion
    ];
```